docs/structure.md

```diff
@@ -1,6 +1,8 @@
 # Orb Structure
 
-Each Orb is packaged and shared as a folder of files that follows a convenient convention for structuring configuration code. Each Orb also has its own namespace to use when invoking its elements. Orbs are authored with similar YAML syntax to the base CircleCI build configuration.
+Each Orb is packaged as a single file named `orb.yml`. Each Orb also has its 
```

Comment:
This doesn't feel quite right -- we're saying that orbs are each a single yaml document, but the platform is agnostic to the file name. This is true above as well, but here it reads particularly oddly.